app/assets/stylesheets/survey_index.scss

```diff
@@ -0,0 +1,32 @@
+@import "colors";
+@import "foundation_and_overrides";
+
+
+
+.button.success.radius.small {
+  margin-left: 170px;
+  margin-right: auto;
+  color: black;
+  background-color: darkseagreen;
+  padding-left: 80px;
+  padding-right: 80px;
+  padding-top: 30px;
+  padding-bottom: 30px;
+  font-size: x-large;
+  font-family: serif;
+  border-radius: 10px;
+  border-width: 2px;
+  border-color: blanchedalmond;
```

Comment:
Color `blanchedalmond` should be written in hexadecimal form as `#ffebcd`<br>Color literals like `blanchedalmond` should only be used in variable declarations; they should be referred to via variable everywhere else.